File: src/learn/helloworld/Helloworld.c
#include <stdio.h>
#include <string.h>

void main()
{
    char user[50];
    printf("Enter your name : ");
    fgets(user, sizeof(user), stdin);
    // scanf("%s", user);    // scanf function read string until reaches space. use fgets function
    printf("Hello world by %s", user);
}

File: src/learn/helloworld/Helloworld.java
package learn.helloworld;

import java.util.Scanner;

public class Helloworld {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter your name :");
        String user = in.nextLine();
        System.out.println("Hello world by " + user);
    }
}